Doc/library/string.rst

```diff
@@ -716,12 +723,14 @@ Replacing ``%x`` and ``%o`` and converting the value to different bases::
    >>> "int: {0:d};  hex: {0:#x};  oct: {0:#o};  bin: {0:#b}".format(42)
    'int: 42;  hex: 0x2a;  oct: 0o52;  bin: 0b101010'
 
-Using the comma or the underscore as a thousands separator::
+Using the comma or the underscore as a digit group separator::
 
    >>> '{:,}'.format(1234567890)
    '1,234,567,890'
    >>> '{:_}'.format(1234567890)
    '1_234_567_890'
+   >>> '{:_o}'.format(1234567890)
```

Comment:
I would have suggested using a hexadecimal example instead of an octal one as people are more familiar with hexadecimal I think, but it's up to you.

Reply:
I replaced it with binary and hexadecimal examples.